src/Stripe.net/Entities/Accounts/Account.cs

```diff
@@ -7,67 +7,148 @@ namespace Stripe
 
     public class Account : StripeEntity<Account>, IHasId, IHasMetadata, IHasObject, IPaymentSource
     {
+        /// <summary>
+        /// Unique identifier for the object.
+        /// </summary>
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        /// <summary>
+        /// String representing the object's type. Objects of the same type share the same value.
+        /// </summary>
         [JsonProperty("object")]
         public string Object { get; set; }
 
+        /// <summary>
+        /// Business information about the account.
+        /// </summary>
         [JsonProperty("business_profile")]
         public AccountBusinessProfile BusinessProfile { get; set; }
 
+        /// <summary>
+        /// The business type.
+        /// Possible values:
```

Comment:
Can we make sure to keep all values on just one line? That's what we do in java already https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Account.java#L32

We should just do exactly the same thing and pattern with `One of ...`